### PR TITLE
Fix up calicoctl configuration docs

### PR DESCRIPTION
**Pull request by caseydavenport**
Our calicoctl configuration docs had some pretty bad formatting, and didn't discuss the k8s backend.

### CODE REVIEW

File: master/reference/calicoctl/setup/etcdv2.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+```yaml
+etcdEndpoints: http://etcd1:2379,http://etcd2:2379,http://etcd3:2379 
+etcdKeyFile: /etc/calico/key.pem 
+etcdCertFile: /etc/calico/cert.pem 
+etcdCACertFile: /etc/calico/ca.pem 
+```
+
+## Environment variables
+
+Configuration may also be specified using the following environment variables.
```
Comment (djosborne) [inline master/reference/calicoctl/setup/etcdv2.md]: Is there an easier way to say this without redundancy? I think the Felix guides used to say something like "Any of the configuration parameters can be passed as Environment variables by capitalizing and adding an underscore before camel cased letters."

Comment (caseydavenport) [inline master/reference/calicoctl/setup/etcdv2.md]: Yep, updated.

**Comment by caseydavenport** [comment]
@djosborne comments addressed!

File: _data/master/navbars/reference.yml [hunk 2]
```diff
@@ -69,8 +69,12 @@ toc:
         path: /reference/calicoctl/resources/workloadendpoint
   - title: Setup
     section:
-      - title: Configuring calicoctl
-        path: /reference/calicoctl/setup/config
+      - title: Overview 
+        path: /reference/calicoctl/setup
+      - title: etcdv2 datastore
```
Comment (djosborne) [inline _data/master/navbars/reference.yml]: Can these be capitalized, like, you know, titles?

**Pull request merged**